dev/infra_server_objects_count_collector.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+usage()
+{
+cat << EOF
+usage: $0 options
+
+This script gets the count of the chef server objects.
+
+OPTIONS:
+   -h      Shows the help message
+   -S      Chef Server url (Eg: https://chef-server.example.com)
```

Comment:
This should not be mandatory, if the user should not provide the value, by default we have to consider the localhost. Think that the end user is running this script from chef infra server machine.